src/kernels/Conv_Winograd_v30_3_1_gfx94x_fp32_f2x3_dilation2.inc

```diff
@@ -2539,40 +2477,464 @@ v_mov_b32_dpp v105, v105 quad_perm:[1,1,1,1] row_mask:0xf bank_mask:0xf
 v_add_co_u32_e32 v104, vcc, v104, v107
 v_add_co_u32_e32 v105, vcc, v105, v106
 v_mov_b32_dpp v106, v37 quad_perm:[0,0,0,0] row_mask:0xf bank_mask:0xf
-v_cmp_ge_u32_e64 s[78:79], v106, s8
-v_sub_co_u32_e64 v106, vcc, v106, s74
+v_cmp_ge_u32_e64 s[2:3], v106, s8
+v_sub_co_u32_e64 v106, vcc, v106, s77
 v_mul_lo_u32 v106, v106, s45
 v_lshlrev_b32_e32 v105, 1, v105
-s_and_b32 s92, 1, s27
-v_add_co_u32_e32 v105, vcc, s92, v105
+s_and_b32 s22, 1, s27
+v_add_co_u32_e32 v105, vcc, s22, v105
 v_lshlrev_b32_e32 v104, 1, v104
-s_and_b32 s92, 1, s26
-v_subrev_co_u32_e32 v104, vcc, s92, v104
-v_mad_i32_i24 v37, v104, s29, v105
+s_and_b32 s22, 1, s26
+v_subrev_co_u32_e32 v104, vcc, s22, v104
+v_mad_i32_i24 v37, v104, s50, v105
 v_add_co_u32_e32 v37, vcc, v37, v106
 v_subrev_co_u32_e32 v62, vcc, 1, v37
-v_add_co_u32_e32 v63, vcc, s29, v37
-v_add_co_u32_e32 v88, vcc, s29, v62
-v_cmp_ge_u32_e64 s[96:97], v105, s29
-s_or_b64 s[94:95], s[96:97], s[78:79]
+v_add_co_u32_e32 v63, vcc, s50, v37
+v_add_co_u32_e32 v88, vcc, s50, v62
+v_cmp_ge_u32_e64 s[98:99], v105, s29
+s_or_b64 s[96:97], s[98:99], s[2:3]
 v_subrev_co_u32_e32 v105, vcc, 1, v105
-v_cmp_ge_u32_e64 s[96:97], v105, s29
-s_or_b64 s[96:97], s[96:97], s[78:79]
-v_cmp_ge_u32_e64 s[92:93], v104, s28
-s_or_b64 s[78:79], s[94:95], s[92:93]
-v_cndmask_b32_e64 v37, v37, -1, s[78:79]
-s_or_b64 s[78:79], s[96:97], s[92:93]
-v_cndmask_b32_e64 v62, v62, -1, s[78:79]
+v_cmp_ge_u32_e64 s[98:99], v105, s29
+s_or_b64 s[98:99], s[98:99], s[2:3]
+v_cmp_ge_u32_e64 s[22:23], v104, s28
+s_or_b64 s[2:3], s[96:97], s[22:23]
+v_cndmask_b32_e64 v37, v37, -1, s[2:3]
+s_or_b64 s[2:3], s[98:99], s[22:23]
+v_cndmask_b32_e64 v62, v62, -1, s[2:3]
 v_add_co_u32_e32 v104, vcc, 1, v104
-v_cmp_ge_u32_e64 s[92:93], v104, s28
-s_or_b64 s[78:79], s[94:95], s[92:93]
-v_cndmask_b32_e64 v63, v63, -1, s[78:79]
-s_or_b64 s[78:79], s[96:97], s[92:93]
-v_cndmask_b32_e64 v88, v88, -1, s[78:79]
+v_cmp_ge_u32_e64 s[22:23], v104, s28
+s_or_b64 s[2:3], s[96:97], s[22:23]
+v_cndmask_b32_e64 v63, v63, -1, s[2:3]
+s_or_b64 s[2:3], s[98:99], s[22:23]
+v_cndmask_b32_e64 v88, v88, -1, s[2:3]
 v_and_b32_e64 v103, v0, 63
-buffer_load_dword v103, v103, s[84:87], 0 idxen
+buffer_load_dword v103, v103, s[88:91], 0 idxen
 s_mov_b64 vcc, s[6:7]
-s_branch 64449
+s_branch 64581
+s_mov_b64 s[98:99], s[84:85]
+s_mov_b32 s97, s87
+v_bfe_u32 v104, s14, 21, 1
+v_sub_co_u32_e64 v104, vcc, v104, 1
+v_cndmask_b32_e32 v105, v63, v37, vcc
+v_cndmask_b32_e32 v106, v88, v62, vcc
+v_readlane_b32 s96, v103, 0
+v_add_f32_e64 v4, v4, s96
+buffer_store_dword v4, v105, s[84:87], 0 idxen
+v_add_f32_e64 v5, v5, s96
+buffer_store_dword v5, v106, s[84:87], 0 idxen
```

Comment:
The `buffer_store_dword` instruction should be substituted with `_buffer_store_dword` macro call. Please check other gfx94x fp32 sources for the same issue.

Reply:
done